### PR TITLE
HPCC-22961 Add Topology server to the System Admin Guide

### PR DESCRIPTION
**Pull request by g-pan**
NOTE: Only lines 340-347 are part of this pull request - 
the other noted sections are result of white-space re-wrap from opening file in different editors. 

## Type of change:
- [x] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Cloud-compatibility
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

## Testing:
 successful integration testing 
https://github.com/g-pan/github-action-dev-build/actions/runs/11731886128

### CODE REVIEW

**Review by JamesDeFabia — APPROVED** [review]
Looks good from my POV.

File: docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml [hunk 1]
```diff
@@ -338,6 +338,13 @@
           Those credentials are then used to authenticate any requests from
           those tools.</para>
         </sect3>
+
+        <sect3 id="TopoSvr">
+          <title>Topology Server</title>
+
+          <para>The topology server is an internal component used by ROXIE.
```
Comment (ghalliday) [inline docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml:345]: ...by ROXIE to keep track of the health of the different roxie processes in a cluster.

We should at least have a minimal comment about what it does.

**Review by ghalliday — COMMENTED** [review]
One comment.
Can anything be done to prevent this reformatting?  Is it an ongoing problem, or a one off because of a different editor used in the past?

File: docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml (hunk 1 above)
Comment (g-pan) [reply docs/EN_US/HPCCSystemAdmin/HPCCSystemAdministratorsGuide.xml]: It was a one-off specific to this file - we use only the one XML editor for any files destined for the repo just for that reason.

**Review by ghalliday — APPROVED** [review]
Looks good, please squash

**Pull request merged**